com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/MeshBuilder.java

```diff
@@ -49,7 +49,7 @@
         for (String t : meshDescBuilder.getTexturesList()) {
             res = BuilderUtil.checkResource(this.project, input, "texture", t);
             taskBuilder.addInput(res);
-            Task<?> embedTask = this.project.createTask(res, TextureBuilder.class);
+            Task<?> embedTask = this.project.createTask(res);
```

Comment:
A possibility to have a cubemap as a texture means that resource in may be build not only with TextureBuilder but also, CubemapBuilder.
So, @JCash you were right, it's better to use `createTask` with one parameter in this case